allure-report-plugin-api/src/main/java/ru/yandex/qatools/allure/plugins/ReportConfig.java

```diff
@@ -25,4 +25,12 @@
     @DefaultValue("%s")
     @Property("allure.tests.management.pattern")
     String getTmsPattern();
+
+    /**
+     * Pattern containing stories tracker base URL and one %s placeholder which will be replaced by story id.
+     * Example: http://example.com/%s and @Stories(value = "story name" id = "SOME-123") will give you http://example.com/SOME-123
+     */
+    @DefaultValue("%s")
```

Comment:
@baev what do you think about defaulting to the link to the documentation here about how to specify the `allure.stories[issues][tests.management].tracker.pattern`? This will simplify finding the docs.